Source/driver/Castro.cpp

```diff
@@ -146,32 +146,33 @@ void
 Castro::variableCleanUp ()
 {
 #ifdef GRAVITY
-  if (gravity != 0) {
-    if (verbose > 1 && ParallelDescriptor::IOProcessor()) {
+  if (gravity != nullptr) {
+    if (verbose >= 1 && ParallelDescriptor::IOProcessor()) {
```

Comment:
I think `verbose > 1` was correct here, this is too much information for the typical user.